cmd/backend.go

```diff
@@ -17,6 +18,16 @@ func main() {
 	zerolog.SetGlobalLevel(zerolog.InfoLevel)
 	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
 
+	// Load environment variables via .env files
+	log.Info().Msg("Attempting to load .env file.")
+	dotenv_err := godotenv.Load()
```

Comment:
How about adding a support for passing a env file name as a cli parameter. would help in switching environments easily without changing the actual env file but just the command that runs the binary.

Reply:
Done